src/main.rs

```diff
@@ -568,8 +568,8 @@ impl Hardware {
 		self.drive_cs_lines();
 
 		// Setup time
-		cortex_m::asm::delay(Self::CS_IO_SETUP_CPU_CLOCKS);
-		
+		cortex_m::asm::delay(1000);
```

Comment:
Should this still be a constant instead of a hardcoded number?

Reply:
Oops, yes. Didn't mean to commit that.

Reply:
Ah, this is undone by a later commit.